railties/test/application/asset_debugging_test.rb

```diff
@@ -86,13 +86,17 @@ class ::PostsController < ActionController::Base ; end
       }
 
       cases.each do |(view_method, tag_match)|
-        app_file "app/views/posts/index.html.erb", "<%= #{view_method} '#{contents}', skip_pipeline: true %>"
+        app_file "app/views/posts/#{view_method}.html.erb", "<%= #{view_method} '#{contents}', skip_pipeline: true %>"
 
         app "development"
 
-        class ::PostsController < ActionController::Base ; end
+        class ::PostsController < ActionController::Base
+          def index
+            render params[:view_method]
+          end
+        end
```

Comment:
Instead of passing the view_method as a param, we could also use define_method to pass it directly:
```suggestion
        class ::PostsController < ActionController::Base ; end
        ::PostsController.define_method(:index) { render view_method }
```

Reply:
I will see if it works well. Definitively it keeps the tests simpler.

Reply:
It works well, I applied it in all the tests, thanks for the idea.

Reply:
Great 🥳